src/wp-content/themes/twentyfourteen/content-featured-post.php

```diff
@@ -9,7 +9,7 @@
 
 ?>
 
-<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
+<article id="post-<?php the_ID(); ?>" <?php post_class(); ?> data-index="<?php echo esc_attr( $GLOBALS['index'] ); ?>">
```

Comment:
Could the `index` be eliminated in favor of something like this?

```suggestion
<article id="post-<?php the_ID(); ?>" <?php post_class(); ?> data-index="<?php echo esc_attr( array_search( get_post(), $wp_query->posts ) + 1 ); ?>">
```

Reply:
That seems to work with `twentyfourteen_get_featured_posts()` instead of `$wp_query->posts`:

```suggestion
<article id="post-<?php the_ID(); ?>" <?php post_class(); ?> data-index="<?php echo esc_attr( array_search( get_post(), twentyfourteen_get_featured_posts() ) + 1 ); ?>">
```

Reply:
+1 to eliminating the global if possible. If not, at least name it something more unique :)

> That seems to work with twentyfourteen_get_featured_posts() instead of $wp_query->posts:

Maybe you need to add `global $wp_query;` to bring it in?

Reply:
`$wp_query->posts` gives the array of posts for the home page, not including featured posts.